src/filters/canI.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+angular
+  .module('openshiftCommonUI')
+  .filter('canI', function(AuthorizationService) {
+    return function(resource, verb, projectName) {
+      return AuthorizationService.canI(resource, verb, projectName);
+    };
+  })
+  .filter('canIAddToProject', function(AuthorizationService) {
+    return function(namespace) {
+      return AuthorizationService.canIAddToProject(namespace);
+    };
+  })
+  .filter('canIDoAny', function(canIFilter) {
```

Comment:
`canIDoAny` is specific to the web console pages. Feel like that should live in origin-web-console.